picard/webservice/throttle.py

```diff
@@ -159,12 +159,9 @@ def adjust(hostkey, slow_down):
                     slow_down: `True` if we encountered intermittent server trouble
                     and need to slow down.
     """
-    def in_backoff_phase(hostkey):
-        return CONGESTION_UNACK[hostkey] > CONGESTION_WINDOW_SIZE[hostkey]
-
     if slow_down:
         _slow_down(hostkey)
-    elif not in_backoff_phase(hostkey):
+    elif CONGESTION_UNACK[hostkey] <= CONGESTION_WINDOW_SIZE[hostkey]:
```

Comment:
Although it was trivial, it was more readable, maybe add a comment here?

Reply:
> Although it was trivial, it was more readable, maybe add a comment here?

Yes, i agree. It was more readable, but a function def was a bit overkilling here imho. I'll add a comment.